docs/containers-auth.json.5.md

```diff
@@ -37,8 +37,28 @@ their accounts on quay.io and docker.io:
 An entry can be removed by using a `logout` command from a container
 tool such as `podman logout` or `buildah logout`.
 
+In addition, credential helpers can be configured for specific registries and the credentials-helper
+software can be used to manage the credentials in a more secure way than depending on the base64 encoded authentication
+provided by `login`.  If the credential helpers are configured for specific registries, the base64 encoded will not be used
+for operations concerning credentials of the specified registries.
+
+When the credential helper is in use on a Linux platform, the auth.json file would contain keys specify the registry domain, and values specify the suffix of the program to use (i.e. everything after docker-credential-).  For example:
```

Comment:
```suggestion
When the credential helper is in use on a Linux platform, the auth.json file would contain keys that specify the registry domain, and values that specify the suffix of the program to use (i.e. everything after docker-credential).  For example:
```
Is there supposed to be a dash at the end of `docker-credential-` in the parens at the end?  I removed it in my suggestion.

Reply:
there should be a dash at the end,  the api will append the config value to this string to look for binary.